src/emerge.cpp

```diff
@@ -206,6 +206,7 @@ EmergeManager::~EmergeManager()
 	delete oremgr;
 	delete decomgr;
 	delete schemmgr;
+	delete biomegen;
```

Comment:
should be deleted first, IIRC it keeps a reference to `biomemgr` internally

Reply:
Oh! Thanks for spotting that.